.github/workflows/windows-ci.yml

```diff
@@ -0,0 +1,56 @@
+name: Fast-DDS Windows CI
+
+on:
+  workflow_dispatch:
+    inputs:
+      label:
+        description: 'ID associated to the workflow. Must univocally identify artifacts.'
+        required: true
+        type: string
+      colcon-args:
+        description: 'Extra arguments for colcon cli'
+        required: false
+        type: string
+      cmake-args:
+        description: 'Extra arguments for cmake cli'
+        required: false
+        type: string
+      ctest-args:
+        description: 'Extra arguments for ctest cli'
+        required: false
+        type: string
+      fastdds_branch:
+        description: 'Branch or tag of Fast DDS repository (https://github.com/eProsima/Fast-DDS)'
+        type: string
+        required: true
+
+  push:
+    branches:
+      - 'master'
+      - '2.11.x'
+      - '2.10.x'
+      - '2.6.x'
```

Comment:
Does it make sense to only leave `2.6.x` here?

Reply:
dcc3314